src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -135,7 +135,7 @@ $3dTiles_Provider.prototype.executeCommand = function executeCommand(command) {
             pnts: this.pntsParse.bind(this),
         };
 
-        return Fetcher.arrayBuffer(url).then((result) => {
+        return Fetcher.arrayBuffer(url, layer.fetchOptions).then((result) => {
```

Comment:
Both are a way to give network usage options, so using the same name (`networkOptions`) seems coherent.

The doc can then explain the valid values depend on the content of the layer.